x-pack/plugins/security_solution/public/common/components/exceptions/helpers.tsx

```diff
@@ -401,7 +421,7 @@ export const getCodeSignatureValue = (
     return codeSignature.map((signature) => {
       return {
         subjectName: signature.subject_name ?? '',
-        trusted: signature.trusted ?? '',
+        trusted: signature.trusted.toString() ?? '',
```

Comment:
Fixes another bug where we were passing in a `boolean` type when the builder expected a string so it was marking it as invalid